front_end/html/summarize_logs.html

```diff
@@ -0,0 +1,158 @@
+
+<form method="post" action="/summarize_logs">
```

Comment:
Please use sentence casing throughout this page so it will be consistent with the rest of the app.